spec/support/expect_script_helper.rb

```diff
@@ -31,10 +32,7 @@ def run_example(code)
         FileUtils.mkdir_p(File.dirname(file))
         File.write(file, code)
       end
-
-      unless system("ruby #{files.last.first} 2>&1")
-        raise "Script exited with status: #{$?.exitstatus}"
-      end
+      system("ruby #{files.last.first} 2>&1")
```

Comment:
before there was not output when a script failed, this makes it show the output + exit status